src/main/java/net/schmizz/sshj/transport/Transport.java

```diff
@@ -238,5 +238,8 @@ long write(SSHPacket payload)
      */
     void die(Exception e);
 
-    KeyAlgorithm getKeyAlgorithm(KeyType keyType) throws TransportException;
+    void setHostKeyAlgorithm(KeyAlgorithm keyAlgorithm);
```

Comment:
`KeyExchanger` has a `TransportImpl` to work with, so there is no need to expose the `set...` methods on the interface. Let's remove them..

Reply:
Done